bindings/c/include/opendal.h

```diff
@@ -539,9 +539,9 @@ struct opendal_result_operator_new opendal_operator_new(const char *scheme,
  *
  * * If the `path` points to NULL, this function panics, i.e. exits with information
  */
-struct opendal_error *opendal_operator_blocking_write(const struct opendal_operator_ptr *ptr,
-                                                      const char *path,
-                                                      struct opendal_bytes bytes);
+struct opendal_error *opendal_operator_write(const struct opendal_operator_ptr *ptr,
```

Comment:
How about renaming `opendal_operator_ptr` to `opendal_operator` instead? We design this name when only operator is a pointer, but now, all struct is a poiner instead. I believe that this renaming make our API more consistent.

And we can call the `opendal_operator` as `op` for short.